### DISCUSSION TITLE
Nuxt3 - Sessions

### DISCUSSION THREAD

**Discussion by mirkofisic** [nuxt/nuxt#16679] [Questions]
What about nuxt3 and session, as far as I notice that api calls now go through the h3 framework, what will happen in that case with session how can I create a session on the server for example for mongodb? Is it even possible to use expressjs/express-session instead of h3?

**Comment by wobsoriano** [comment]
This might be worth checking https://github.com/nuxt/framework/discussions/1062

**Comment by christopher4lis** [comment]
> Is it even possible to use expressjs/express-session instead of h3?

This is a great question which I'd also like to know the answer to. Being able to implement auth for a Nuxt 3 app would immediately make it production ready in my eyes. The current nitro features like route generation are already so clutch, just missing the session and auth pieces it seems.
